src/figuresClasses.ts

```diff
@@ -1,19 +1,76 @@
 export interface Figure {
+  color: 'red' | 'green' | 'blue';
+  shape: string;
+  getArea: () => number;
+}
```

Comment:
it is better to write exactly shapes
also, I think it is good idea to put colors and shapes in the separated types